src/main/java/com/google/sps/lib/validator/SubmitJobValidator.java

```diff
@@ -24,7 +24,7 @@
  */
 public class SubmitJobValidator {
   private boolean isAlphabetic(String value) {
-    return Pattern.matches("[a-zA-Z]+", value);
+    return Pattern.matches("[a-zA-Z0-9 ,/.-]+", value);
```

Comment:
This method is called `isAlphabetic`. And you are validating this correct? Add a java test for the change as well.